cmds/webboot/webboot_test.go

```diff
@@ -105,3 +109,132 @@ func TestDownloadByBookmarkOption(t *testing.T) {
 		t.Errorf("Fail to remove test file, error msg: %+v", err)
 	}
 }
+
+func TestGetCachedIsos(t *testing.T) {
+	isos := getCachedIsos("./testdata/")
+	if len(isos) != 2 {
+		t.Errorf("Miss some cached iso, expect 2, find %v", len(isos))
+	}
+
+	for _, iso := range isos {
+		t.Log(iso)
+		if _, err := os.Stat(iso.path); err != nil {
+			t.Errorf("Fail to find cached file, error msg: %+v", err)
+		}
+
+	}
+}
+
+func TestInstallCachedISOOption(t *testing.T) {
+	cachedISO := []*CachedISO{
+		&CachedISO{
+			label: "test cached iso1",
+			group: "test group 1",
+		},
+		&CachedISO{
+			label: "test cached iso2",
+			group: "test group 1",
+		},
+		&CachedISO{
+			label: "test cached iso3",
+			group: "test group 2",
+		},
+		&CachedISO{
+			label: "test cached iso4",
+			group: "test group 2",
+		},
+	}
+
+	for _, tt := range []struct {
```

Comment:
This is really neat. This it the first time, EVERY, in this project, and its predecessor, that we've ever had UI testing. Very nice :-)